students/johnpharmd/lesson04/mailroom2.py

```diff
@@ -0,0 +1,134 @@
+#!/usr/bin/env python3
+import datetime
+
+# Data
+donors_amts = {'Gates': [('Mr.', ), 150000, 3], 'Brin': [('Mr.', ), 150000, 3], 
+'Cerf': [('Mr.', ), 50000, 2], 'Musk': [('Mr.', ), 100000, 1], 
+'Berners-Lee': [('Mr.', ), 50000, 2], 
+'Wojcicki': [('Ms.', ), 125000, 1], 'Avey': [('Ms.', ), 200000, 2]}
```

Comment:
this and some of the string multi-line statements should be one indent or aligned with first paren, example:
```
donors_amts = {'Gates': [('Mr.',), 150000, 3], 'Brin': [('Mr.',), 150000, 3],
               'Cerf': [('Mr.',), 50000, 2], 'Musk': [('Mr.',), 100000, 1],
               'Berners-Lee': [('Mr.',), 50000, 2],
               'Wojcicki': [('Ms.',), 125000, 1], 'Avey': [('Ms.',), 200000, 2]}
```